book-example/src/format/config.md

```diff
@@ -45,6 +48,20 @@ src = "my-src"  # the source files will be found in `root/my-src` instead of `ro
 build-dir = "build"
 ```
 
+### Build options
+
+This controls the build process of your book.
+
+- **create-missing:** By default, any missing files specified in `SUMMARY.md`
+  will be created when the book is built. If this is `false` then the build
```

Comment:
Would it be worth explicitly saying that this defaults to `true`? So something like *"... created when the book is built (i.e. `create-missing = true`)"*.

Reply:
Yes, that makes sense to me.